src/app/find/mapPage/mapPage.css

```diff
@@ -0,0 +1,86 @@
+@value primaryColor from "../../../colors.css";
+
+.resultsBar{
+	background-color: #ffffff;
+	position: absolute;
+	left: 0;
+	top: 0;
+	right: 0;
+	z-index: 2;
+	padding: 1rem 0.5rem 1rem 0.5rem !important;
+	box-shadow: 0px 8px 8px rgba(0,0,0, 0.33);
+}
+
+.serviceCategory{
+	font-size: .80rem;
+	letter-spacing: 0.3px;
+	color: #ffffff;
+	width: 98%;
+	text-transform: capitalize;
+	font-weight: 600;
+	line-height: 1rem;
+	padding-bottom: 0.33rem;
+
+}
+
+.categoriesContainer{
+	box-shadow: 0px -8px 4px rgba(0,0,0,.25);
+	border-radius: 12px 12px 0px 0px;
+	position: absolute;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	background-color: #FFFFFF;
+	z-index: 2;
+	display: flex;
+	flex-direction: column;
+	align-content: center;
+	align-items: center;
+}
+
+.categoriesTitle {
+	padding: 1rem 1rem 0.5rem 1rem;
+	font-size: 1.1rem !important;
+	font-weight: bold;
+}
+
+.buttonsContainer{
+	display: flex;
+	justify-content: center;
+	width: 100%;
+}
+
+.categoryButton{
```

Comment:
On a desktop, the category buttons look absolutely huge now (cover the majority of the screen's height). Was that intentional?

Reply:
Also, is it just me, or is the shower icon much higher than the other buttons?

Reply:
It's not optimized for desktop, at all.

Reply:
Oh OK, cool.
I think the shower icon thing was happening in the mobile view too though.